demo/apisix/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM apache/apisix:3.2.0-debian
```

Comment:
## Why do we need it?

1) Apisix and Plugin Runner must be run in the same container
2) If we use `apache/apisix:3.2.0-debian`, we got an error when running. Ref (https://github.com/apache/apisix-java-plugin-runner/issues/165)
3) Docker image for this does not exist in apisix dockerhub